plenoptic/metric/perceptual_distance.py

```diff
@@ -399,62 +415,86 @@ def nlpd(IM_1, IM_2):
 
     Parameters
     ----------
-    IM_1: torch.Tensor
-        image, (1 x 1 x H x W)
-    IM_2: torch.Tensor
-        image, (1 x 1 x H x W)
-
+    IM_1: torch.Tensor of shape (batch, channel, height, width)
```

Comment:
doesn't have to happen in this PR, but we need to go through all these argument names and make them consistent: we've got `img1`, `im_1` and `IM_1` all in this file.